lib/resource-providers/vpc.ts

```diff
@@ -30,8 +32,15 @@ export class VpcProvider implements ResourceProvider<ec2.IVpc> {
             // Network routing for the public subnets will be configured to allow outbound access directly via an Internet Gateway.
             // Network routing for the private subnets will be configured to allow outbound access via a set of resilient NAT Gateways (one per AZ).
             vpc = new ec2.Vpc(context.scope, id + "-vpc");
+            if (this.secondaryCidrId) {
+                new ec2.CfnVPCCidrBlock(context.scope, id + "-secondaryCidr", {
+                    vpcId: vpc.vpcId,
+                    cidrBlock: this.secondaryCidrId});
+            } else {
```

Comment:
Unsure about this else block. I am concerned that if customers are already running blueprints, they will all of a sudden get a secondary cidr block, which may unexpected. Let's drop it and just provide and approach to `opt-in` if they want to.

Reply:
Sure. I fixed it. Make sense.